File: DSA/Angel_500119630/Day_6_Question_2.java
public class Day_6_Question_2{
    public static void main(String[] args) {
        int[] nums = {1, 2, 3, 4}; 
        System.out.println(isStrictlyIncreasing(nums));
    }

    public static boolean isStrictlyIncreasing(int[] nums) {
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] <= nums[i - 1]) {
                return false;
            }
        }
        return true;
    }
}

File: DSA/Shubham_590016506/Day_6_Question_2.c
#include <stdio.h>
#include <stdbool.h>

bool isStrictlyIncreasing(int* nums, int numsSize) {
    for (int i = 1; i < numsSize; i++) {
        if (nums[i] <= nums[i - 1]) return false;
    }
    return true;
}

int main() {
    int nums1[] = {1, 2, 3, 4};
    int nums2[] = {1, 2, 2, 3};

    printf("%s\n", isStrictlyIncreasing(nums1, 4) ? "true" : "false");
    printf("%s\n", isStrictlyIncreasing(nums2, 4) ? "true" : "false");
    return 0;
}